File: src/app/@core/data/product-group.ts
import { ProductGroupDetail } from './product-group-detail';

export interface ProductGroup {
    id: number;
    companyCode: string;
    productGroupCode: string;
    customerCode: string;
    productGroupName: string;
    groupType: string;
    description: string;
    createdBy?: string;
    createdDateTime?: Date;
    lastUpdatedBy?: string;
    lastUpdatedDateTime?: Date;
    deleted: boolean;
    productGroupDetail: ProductGroupDetail[];
}

File: inventoryserver/inventoryserver/Models/ProductGroup.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace inventoryserver.Models
{
  public partial class ProductGroup
  {
    public int Id { get; set; }
    public string CompanyCode { get; set; }
	  public string ProductGroupCode { get; set; }
	  public string CustomerCode { get; set; }
	  public string ProductGroupName { get; set; }
	  public string GroupType { get; set; }
	  public string Description { get; set; }
	  public string CreatedBy { get; set; }
	  public DateTime? CreatedDateTime { get; set; }
	  public string LastUpdatedBy { get; set; }
	  public DateTime? LastUpdatedDateTime { get; set; }
	  public bool Deleted { get; set; }
  }
}
